src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -323,13 +327,34 @@ std::map<std::string, ov::Any> properties_to_any_map(const std::map<std::string,
         } else if (property.first == ov::hint::model.name()) {
             auto model = Common::utils::convert_to_model(property.second);
             properties_to_cpp[property.first] = std::static_pointer_cast<const ov::Model>(model);
+        } else if (property.first == ov::intel_auto::devices_utilization_threshold.name() &&
+                   py::isinstance<py::dict>(property.second)) {
+            std::string value = convert_dict_to_string(property.second);
+            properties_to_cpp[property.first] = value;
         } else {
             properties_to_cpp[property.first] = Common::utils::py_object_to_any(property.second);
         }
     }
     return properties_to_cpp;
 }
 
+std::string convert_dict_to_string(const py::object& obj) {
+    if (!py::isinstance<py::dict>(obj)) {
+        return py::str(obj).cast<std::string>();
```

Comment:
why it's casted to string if no dict by default?

Reply:
maybe a mistake. Updated.